lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -18,17 +18,14 @@ class ReportRenderer {
   /**
    * @param {!DOM} dom
    * @param {!CategoryRenderer} categoryRenderer
-   * @param {?ReportUIFeatures=} uiFeatures
    */
-  constructor(dom, categoryRenderer, uiFeatures = null) {
+  constructor(dom, categoryRenderer) {
```

Comment:
`reportUIFeatures` is removed from the `report-renderer` constructor, which should make devtools subclassing without it slightly easier cc @paulirish